packages/blade/src/components/Amount/__tests__/mock.ts

```diff
@@ -0,0 +1,15 @@
+import type { AmountProps } from '../Amount';
+
+export const AMOUNT_SUFFIX_TEST_SET: {
+  value: number;
+  suffix: AmountProps['suffix'];
+  output: string;
+  locale?: string;
+}[] = [
+  { value: 1000000.22, suffix: 'humanize', output: '1 Mio.', locale: 'de-DE' },
```

Comment:
shouldn't we also have currency symbols in these mocks?

Reply:
Currency symbol was little redundant here as formatting was the core part being driven by locale and was being checked already. But I've still included it now in the test case suite.